pkgs/servers/nextcloud/packages/apps/recognize.nix

```diff
@@ -0,0 +1,109 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  nodejs,
+  python311,
+  util-linux,
+  ffmpeg,
+
+  # Current derivation only suports linux-x86_64 (contributions welcome, without libTensorflow builtin webassembly can be used)
+  useLibTensorflow ? stdenv.isx86_64 && stdenv.isLinux,
```

Comment:
What is the alternative to this? Will it be downloaded at runtime? Or just fail?
Perhaps it's also fine to just support x86_64-linux for now with meta.platforms.

Reply:
If you set it to false only the wasm version will work, (as the patch phase disables the installation). TensorFlow only supports  64 bit machines anyway (it does also support darwin, but I have no such device to test with). 

It should work on all platforms, just slower in WASM mode :sweat_smile: